CHANGELOG.md

```diff
@@ -1,23 +1,81 @@
-Changes by Version
-==================
-
 ### A message to people of Russia 🇷🇺
 
 If you currently live in Russia, please read [this message](./_To_People_of_Russia.md).
 
+Changes by Version
+==================
+
+<details>
+<summary>next release template</summary>
+
 next release (yyyy-mm-dd)
 -------------------
 
 ### Backend Changes
 
-#### ⛔ Breaking Changes
+run `make changelog` to generate content
 
-#### New Features
+### UI Changes
 
-#### Bug fixes, Minor Improvements
+...
+
+</details>
+
+1.51.0 (2023-11-04)
+-------------------
+
+### Backend Changes
+
+#### ✨ New Features:
+
+* Feat: add sampling store support to badger ([@slayer321](https://github.com/slayer321) in [#4834](https://github.com/jaegertracing/jaeger/pull/4834))
+* Feat: add span adjuster that moves some otel resource attributes to span.process ([@james-ryans](https://github.com/james-ryans) in [#4844](https://github.com/jaegertracing/jaeger/pull/4844))
+* Add product/file version in windows executables ([@ResamVi](https://github.com/ResamVi) in [#4811](https://github.com/jaegertracing/jaeger/pull/4811))
+
+#### 🐞 Bug fixes, Minor Improvements:
+
+* Fix dependency policy and add to security-insights.yml ([@jkowall](https://github.com/jkowall) in [#4907](https://github.com/jaegertracing/jaeger/pull/4907))
+* Add reload interval to otel server certificates ([@james-ryans](https://github.com/james-ryans) in [#4898](https://github.com/jaegertracing/jaeger/pull/4898))
+* Feat: add blackhole storage, for benchmarking ([@yurishkuro](https://github.com/yurishkuro) in [#4896](https://github.com/jaegertracing/jaeger/pull/4896))
+* Add otel resource detector to jaeger components ([@james-ryans](https://github.com/james-ryans) in [#4864](https://github.com/jaegertracing/jaeger/pull/4864))
+* Fix batchprocessor to set correct span format flags ([@k0zl](https://github.com/k0zl) in [#4796](https://github.com/jaegertracing/jaeger/pull/4796))
+* Expose collector ports in docker images ([@arunvelsriram](https://github.com/arunvelsriram) in [#4810](https://github.com/jaegertracing/jaeger/pull/4810))
+
+#### 🚧 Experimental Features:
+
+* Exp(jaeger-v2): simplify all-in-one configuration ([@yurishkuro](https://github.com/yurishkuro) in [#4875](https://github.com/jaegertracing/jaeger/pull/4875))
+* Exp: support primary and archive storage ([@yurishkuro](https://github.com/yurishkuro) in [#4873](https://github.com/jaegertracing/jaeger/pull/4873))
+* Feat(jaeger-v2): create default config for all-in-one ([@yurishkuro](https://github.com/yurishkuro) in [#4842](https://github.com/jaegertracing/jaeger/pull/4842))
+
+#### 👷 CI Improvements:
+
+* Ci: split the install-tools into test/build groups ([@MeenuyD](https://github.com/MeenuyD) in [#4878](https://github.com/jaegertracing/jaeger/pull/4878))
```

Comment:
The `Ci: ` prefixes seem a little superfluous, and similarly for other sections.

Reply:
Yeah, although not unusual in other projects. Now that we switched to using labels we should pay closer attention to PR titles to be exactly what we want to show up in the changelog.